CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ This changelog is initialized in release 1.0.0
 
 ## [Unreleased]
 
+### Added
+* Cursor method to repository
```

Comment:
Could you add a new release (2.8.0) tag above this in this PR? Then I can do an immediate release after merge.

Reply:
Done!